src/dashboard/src/requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
 -r test.txt
 
 ipython
+ipdb
```

Comment:
Cool, great idea. Once you import ipdb I would try to start a new container instead of use an existing one:

    $ docker-compose run --rm --publish 9999:8000 archivematica-storage-service

The port `8000` should be mapped to your host as `9999`. We've configured Gunicorn so it starts 4 workers, that could be a problem when you're debugging. If so, you could pass an extra `-e SS_GUNICORN_WORKERS=1` to `docker-compose run`.